ndcube/asdf/converters/ndcollection_converter.py

```diff
@@ -0,0 +1,25 @@
+from asdf.extension import Converter
+
+
+class NDCollectionConverter(Converter):
+    tags = ["tag:sunpy.org:ndcube/ndcube/ndcollection-*"]
+    types = ["ndcube.ndcollection.NDCollection"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from ndcube.ndcollection import NDCollection
+
+        key_value_pairs = list(zip(node["keys"], node["value"]))
+        aligned_axes = list(node.get("aligned_axes").values())
+        aligned_axes = tuple(tuple(lst) for lst in aligned_axes)
+        return NDCollection(key_value_pairs, meta=node.get("meta"), aligned_axes=aligned_axes)
+
+    def to_yaml_tree(self, ndcollection, tag, ctx):
+        node = {}
+        node["keys"] = tuple(ndcollection.keys())
+        node["value"] = tuple(ndcollection.values())
```

Comment:
Shouldn't this be `node['values']`? Or at least, wouldn't it be more intuitive if it were?